pkgs/by-name/te/technitium-dns-server/package.nix

```diff
@@ -1,39 +1,60 @@
 {
   lib,
-  stdenvNoCC,
-  fetchurl,
-  makeWrapper,
-  dotnet-sdk_8,
+  buildDotnetModule,
+  fetchFromGitHub,
+  dotnetCorePackages,
   nixosTests,
 }:
-stdenvNoCC.mkDerivation rec {
+let
+  technitium-library = buildDotnetModule rec {
+    pname = "TechnitiumLibrary";
+    version = "40a7bc7a52cf5d2c371215ea7f5f6fdf71dc1d9c";
+
+    src = fetchFromGitHub {
+      owner = "TechnitiumSoftware";
+      repo = "TechnitiumLibrary";
+      rev = version;
+      hash = "sha256-FNCZH8LZXYyrz5SRIV8P7MruN+t6Qvl96Aa73q3InMI=";
+      name = "${pname}-${version}";
+    };
+
+    dotnet-sdk = dotnetCorePackages.sdk_8_0;
+
+    nugetDeps = ./library-nuget-deps.nix;
```

Comment:
How about just removing `library-nuget-deps.nix` and use an empty list? Or if possible, removing the attribute `nugetDeps` completely ?

Reply:
As far as I understand, nugetDeps is mandatory and may not be null (https://nixos.org/manual/nixpkgs/stable/#packaging-a-dotnet-application).

When writing the PR, I had a look at other dotnet-modules and there is always (?) a deps.nix. It might be empty (for example https://github.com/NixOS/nixpkgs/tree/83a67322b1084c68c09fbc1f1e608073ec2c594a/pkgs/applications/audio/famistudio). 

As I assume that it's not possible to remove nugetDeps, I'd prefer an "empty" file and not a derivation.